src/util/badgeControl.js

```diff
@@ -0,0 +1,25 @@
+L.Control.Badge = L.Control.extend({
```

Comment:
Excellent!

Reply:
Let's add this badge functionality to Readme as well. Telling that this is automatically there if you use 
` L.LayerGroup.EnvironmentalLayers().addTo(map);`

otherwise using the above function (which calls listeners) you can add!

Reply:
Okay 👍